evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -273,7 +273,7 @@ def test_make_pipeline_no_column_names(input_type, problem_type):
 def test_make_pipeline_text_columns(input_type, problem_type):
     X = pd.DataFrame({"numerical": [1, 2, 3, 1, 2],
                       "categorical": ["a", "b", "a", "c", "c"],
-                      "text": ["string one", "another", "text for a column", "text string", "hello world"]})
+                      "text": ["string one", "another", "text for a column, this should be a text column!!", "text string", "hello world"]})
```

Comment:
Making this longer so that WW automatically detects its a text column. Could also just use `infer_feature_types` heh